frontend/public/components/utils/file-input.tsx

```diff
@@ -94,7 +111,7 @@ export class FileInput extends React.Component<FileInputProps, FileInputState> {
                 }
                 className="pf-c-form-control co-file-dropzone__textarea"
                 onChange={this.onDataChange}
-                value={this.props.inputFileData}
+                value={fileIsBinary ? fileIsBinaryMsg : this.props.inputFileData}
```

Comment:
I think it would be better to hide the textarea instead of putting the message inside the textarea. (For one, there's no way to tell that the message is not the contents of the file you picked.)

Reply:
hide text area, and add an info alert that the text contains non-printable characters.